docs/5.x/tests-php.md

```diff
@@ -354,10 +354,10 @@ any change in the output is actually expected.
 If they are not expected, determine the cause of the change and fix it in a new commit. If the changes are expected,
 then you should update the expected system files accordingly. To compare and update the expected system files, follow these steps:
 
-* Find out the Travis build number by opening the Travis run for your pull request. The build number is typically a 5 or 6 digit number and has a leading hash character. For example when the build is `#45511`, then `45511` is the build number.
+* Find out the GitHub build number by opening the GitHub action run for your pull request. The build number is typically a 10 digit number.
```

Comment:
@sgiehl in a URL like this, which one is it? `actions/runs/4070393437/jobs/7011193382`

could we specify that a bit more clearly? i tried both but it's not really working yet

Reply:
@tsteur would that explain it better:
```suggestion
* Find out the GitHub build number by opening the GitHub action run for your pull request. The build number is typically a 10 digit number and is part of the URL (follows on `/runs/`). When looking at a certain job result a typical URL looks like this `/actions/runs/4070393437/jobs/7011193382`. The build number in this case would be `4070393437`.
```

Reply:
That's great 🎉